packages/react-file-type-icons/CHANGELOG.json

```diff
@@ -1,70 +1,70 @@
 {
-  "name": "@fluentui/file-type-icons",
+  "name": "@fluentui/react-file-type-icons",
   "entries": [
     {
       "date": "Mon, 26 Oct 2020 12:39:59 GMT",
-      "tag": "@fluentui/file-type-icons_v8.0.0-beta.1",
+      "tag": "@fluentui/react-file-type-icons_v8.0.0-beta.1",
```

Comment:
I think we should generally leave the existing entries as they are since we didn't change the existing tags and IIRC these entries won't be used again when generating release notes.

Reply:
done